app/src/main/java/com/io/movio/ui/movielist/MovieListViewModel.kt

```diff
@@ -15,7 +16,7 @@ class MovieListViewModel : ViewModel() {
 
     init {
         viewModelScope.launch {
-            val result = GetMoviesUseCase().execute(Unit)
+            val result = GetMoviesUseCase(MovieRepository).execute(Unit)
```

Comment:
In the GetMovieById use case you provide the repository directly in the constructor, maybe you can do the same for this use case so your code will be consistent